Bottom: 49db3860102c73fb972d401aa0dad8ce194fa929
Top:    e9d69d3dac916f028eedd6161ac062467a1a07e8
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-22 09:33:10 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/Vagrantfile b/Vagrantfile
index b2f0fba5..a124020d 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -60,8 +60,8 @@ Vagrant.configure("2") do |config|
             id: "ssh"
 
         main.vm.synced_folder "/mnt/vm/bacula-restores",
-            "/mnt/bacula-restores",
-            id: "bacula-restores",
+            "/mnt/bacula-storage",
+            id: "bacula-storage",
             owner: "root",
             group: "root",
             mount_options: ["ro"]
diff --git a/ansible-playbook/group_vars/docker-hosts/repos.yml b/ansible-playbook/group_vars/docker-hosts/repos.yml
index ddb6a36b..cc4d9cbf 100644
--- a/ansible-playbook/group_vars/docker-hosts/repos.yml
+++ b/ansible-playbook/group_vars/docker-hosts/repos.yml
@@ -4,6 +4,7 @@ buildbot_image_repo: https://github.com/AndrewCooper/docker-buildbot-master.git
 buildbot_worker_ntel_image_repo: https://github.com/AndrewCooper/docker-buildbot-worker-ntel.git
 buildbot_worker_ptxdist_image_repo: https://github.com/AndrewCooper/docker-buildbot-worker-ptxdist.git
 exim4_image_repo: https://github.com/AndrewCooper/docker-exim4.git
+gitlab_image_repo: 'https://github.com/AndrewCooper/docker-gitlab.git'
 mantisbt_image_repo: https://github.com/AndrewCooper/docker-mantisbt.git
 openldap_image_repo: https://github.com/AndrewCooper/docker-openldap.git
 openssl_image_repo: https://github.com/AndrewCooper/docker-openssl.git
